mining.md

```diff
@@ -119,17 +119,27 @@ func VerifyBlock(blk Block) {
         Fatal("invalid block signature")
     }
     
-    // 2. Verify ParentWeight
+    // 2. Verify Timestamp
+    // first check that it is not in the future
+    if blk.GetTime() > nAdjustedTime {
```

Comment:
where is nAdjustedTime defined?

Reply:
In the BTC code base :b 

If you like the concept, I'd just say implementation-dependent but could simply be a call to an NTP server...

Reply:
i would be happy if this just said `blk.GetTime() > time.Now()`

Reply:
ok, I do think asking folks to refer to NTP would help though, no. 

Is that a broader point we can make to miners somwhere (ie sync your clocks to network time)?